cookbooks/aws-parallelcluster-computefleet/test/controls/custom_parallelcluster_node_spec.rb

```diff
@@ -0,0 +1,28 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+control 'custom_parallelcluster_node_installed' do
+  file_cache_path = "/tmp/kitchen/cache"
+
+  title "custom aws-parallelcluster-node should have been installed in the node virtualenv"
+  only_if { !os_properties.redhat_ubi? }
```

Comment:
Why not ?

Reply:
Good question. I actually started working from [node_spec.rb](https://github.com/aws/aws-parallelcluster-cookbook/blob/b4ef8f5c0b586af0f1e306ce5ea81700a192321b/cookbooks/aws-parallelcluster-computefleet/test/controls/node_spec.rb), which currently has that if clause as well. I'm not sure why: I'll try to remove it and see if it breaks on docker or RHEL.